src/main/java/com/speedment/internal/core/code/manager/EntityManagerImplTranslator.java

```diff
@@ -206,13 +206,13 @@ private Method defaultReadEntity(File file) {
                     .append("resultSet.")
                     .append("get")
                     .append(mapping.getResultSetMethodName(dbms()))
-                    .append("(\"").append(c.getName()).append("\")");
+                    .append("(").append(c.getOrdinalPosition()).append(")");
             } else {
                 sb
                     .append("get")
                     .append(mapping.getResultSetMethodName(dbms()))
                     .append("(resultSet, ")
-                    .append("\"").append(c.getName()).append("\")");
+                    .append(c.getOrdinalPosition()).append(")");
```

Comment:
If someone is reordering the columns, it will still work because the columns are explicitly selected by the Manager: e.g:
"select id, name, age from user" so if age and name is swapped and then we regenerate, it will still work. The old code will also still work because the order is decided by the manager, not the order they appear in the table.